nexus/caller/starter/main.go

```diff
@@ -1,3 +1,4 @@
+// @@@SNIPSTART startercaller-nexus
```

Comment:
You'll want to prefix all of these snippets with `samples-go-` as done for the rest of the snippets in this repo as snipsync doesn't use the repo name to prefix the snippet name.

Reply:
Sounds good.